tests/integ/test_data_source_api.py

```diff
@@ -105,15 +105,22 @@
 ORACLEDB_TEST_EXTERNAL_ACCESS_INTEGRATION = "snowpark_dbapi_oracledb_test_integration"
 
 
+@pytest.mark.parametrize(
+    "input_type, input_value",
+    [
+        ("table", SQL_SERVER_TABLE_NAME),
+        ("query", f"SELECT * FROM {SQL_SERVER_TABLE_NAME}"),
+        ("query", f"(SELECT * FROM {SQL_SERVER_TABLE_NAME})"),
+    ],
+)
 @pytest.mark.parametrize("fetch_with_process", [True, False])
-def test_dbapi_with_temp_table(session, caplog, fetch_with_process):
+def test_basic_sql_server(session, caplog, fetch_with_process, input_type, input_value):
+    input_dict = {
+        input_type: input_value,
+        "fetch_with_process": str(fetch_with_process),
```

Comment:
I don't think you need to stringfy fetch_with_process, or would it go wrong with boolean input?